src/garage/experiment/local_runner.py

```diff
@@ -117,10 +116,12 @@ def __init__(self, snapshot_config, max_cpus=1):
                                         snapshot_config.snapshot_mode,
                                         snapshot_config.snapshot_gap)
 
-        if max_cpus > 1:
-            # pylint: disable=import-outside-toplevel
-            from garage.sampler import singleton_pool
-            singleton_pool.initialize(max_cpus)
+        parallel_sampler.initialize(max_cpus)
```

Comment:
Shouldn't this be placed in the TF Runner?

Reply:
Is this actually necessary? Can't we afford to start the sampler in `setup`, once we actually know the sampler?

Reply:
We can't set it in `setup()` at least for all tf algorithms due to tf sessions.